pkg/services/ldap/settings.go

```diff
@@ -135,9 +138,28 @@ func readConfig(configFile string) (*Config, error) {
 		}
 	}
 
+	for _, server := range result.Servers {
```

Comment:
Can we add some tests for it?

Reply:
Agreed. Could you add a test @JonasDeGendt?